coredb-operator/tests/integration_tests.rs

```diff
@@ -146,13 +135,31 @@ mod test {
             .unwrap();
         println!("{}", result.stdout.clone().unwrap());
         assert!(result.stdout.clone().unwrap().contains("CREATE TABLE"));
+
+        // Assert table 'customers' exists
         let result = coredb_resource
             .psql("\\dt".to_string(), "postgres".to_string(), client.clone())
             .await
             .unwrap();
         println!("{}", result.stdout.clone().unwrap());
         assert!(result.stdout.clone().unwrap().contains("customers"));
 
+        // TODO(ianstanton) we need to properly wait for 'postgis' extension to be created
+        thread::sleep(Duration::from_millis(500));
```

Comment:
I think this is possible to do with a 'condition', like we have in the case of is_pod_ready, but not necessary right away for a test